other/imagepullpolicy-always.yaml

```diff
@@ -12,8 +17,10 @@ spec:
         kinds:
         - Pod
     validate:
-      message: "The imagePullPolicy must be set to `Always` for all containers when a tag other than `latest` is used."  
+      message: >-
+        The imagePullPolicy must be set to `Always` when the tag `latest` is used.
       pattern:
         spec:
           containers:
-          - imagePullPolicy: Always
\ No newline at end of file
+          - (image): "*:latest | !*:*"
```

Comment:
This expression will only work in Kyverno 1.3.0. For reasons like this, I'm proposing we implement a new annotation (see wiki) which notes the minimum version of Kyverno needed to correctly implement a policy.